moveit_commander/test/python_moveit_commander_ros_namespace.test

```diff
@@ -0,0 +1,5 @@
+<launch>
+  <include ns="test_ros_namespace" file="$(find moveit_resources_fanuc_moveit_config)/launch/test_environment.launch"/>
+  <test ns="test_ros_namespace" pkg="moveit_commander" type="python_moveit_commander_ros_namespace.py" test-name="python_moveit_commander_ros_namespace"
+        time-limit="300" args=""/>
```

Comment:
```suggestion
  <!-- PlanningSceneInterface can connect to namespaced moveit configuration -->
  <group ns="test_ros_namespace">
    <include file="$(find moveit_resources_fanuc_moveit_config)/launch/test_environment.launch"/>
    <test pkg="moveit_commander" type="python_moveit_commander_ros_namespace.py" test-name="python_moveit_commander_ros_namespace"
          time-limit="300" args=""/>
  </group>
```

You might have to clean up the indentation here because I typed this in the edit box...